app/uk/gov/hmrc/homeofficesettledstatusstubs/stubdata/DemoStubData.scala

```diff
@@ -22,7 +22,7 @@ import java.time.LocalDate
 
 object DemoStubData extends DataSet {
 
-  val lawrenceVelazquez = StatusCheckResult(
+  val lawrenceVelazquez: StatusCheckResult = StatusCheckResult(
```

Comment:
All these names should really be changed to follow the correct test data in the open guidance : https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open

Happy to create a separate ticket for this if you want?

Reply:
done